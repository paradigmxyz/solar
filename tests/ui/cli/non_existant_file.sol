//@ compile-flags: doesnotexist.sol
//@ error-in-other-file: file doesnotexist.sol not found
// check-fail
