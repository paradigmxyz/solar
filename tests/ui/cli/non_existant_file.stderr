error: file doesnotexist.sol not found

error: aborting due to 1 previous error

